oracle/oracle-how-to/use-bastion-to-access-VM.rst

```diff
@@ -91,45 +137,155 @@ When the bastion plugin shows up as *Running* in the cloud console, you should s
+        where
+
+        * ``<path_to_id_rsa.pub_key>`` should be replaced with a public SSH key that you want injected into the bastion and the VM (you must have access to the corresponding private key)
+        * ``<virtual_machine_ocid>`` should be replaced with the OCID of the VM to be accessed and 
+        * 'ubuntu' should be replaced with the username that you would like to use to SSH into your VM ('ubuntu' is the default username in Ubuntu images)
 
-.. image:: use-bastion-to-access-VM-images/4_create_session.png
+.. note::
 
+    The SSH key that you use here, will be temporarily added to ``.ssh/authorized_keys`` in both the bastion and the VM. So when you try to access the VM in the next step, you'll have to provide your private key twice -  once to SSH into the bastion, and then again within a proxy command to SSH into the VM.
 
 
 Access the VM
 -------------
 
-When the session is created, use the *Copy SSH command* from the 3 dots menu to get a command similar to:
+You can access the VM using either the console or the CLI.
+
+.. tabs::
+
+    .. group-tab:: Using console
+
+        When the session is created, use the *Copy SSH command* from the 3 dots menu to get a command similar to:
+
+        .. code::
+
+            ssh -i <privateKey> -o ProxyCommand="ssh -i <privateKey> -W %h:%p -p 22 ocid1.bastionsession.oc1.sa-saopaulo-1.amaaaaaaniwq6sya34ss2bksayws7o4lovrtmvxg3e3baqb64vxw7nz447kq@host.bastion.sa-saopaulo-1.oci.oraclecloud.com" -p 22 ubuntu@10.0.0.86
+
+        This command will create a tunnel through the public hostname of the bastion to the private IP address of your VM. Replace ``<privateKey>`` with the path of your private key, and you should have SSH access to your VM using the bastion.
+
+    .. group-tab:: Using CLI
+
+        Check if your managed SSH session is in the 'ACTIVE' state using:
+
+        .. code::
+
+            oci bastion session list --bastion-id <bastion_ocid> \
+                        --session-lifecycle-state ACTIVE \
+                        --sort-order asc --all --query "data[0].id" --raw-output
+        
+        The command displays the OCID of all the managed SSH sessions in the bastion that are in the 'ACTIVE' state. You might have to repeat the command a few times till your session becomes 'ACTIVE'. An 'ACTIVE' session indicates that you can now connect to your VM.
+
+        Using the managed session OCID obtained above, run:
+
+        .. code::
+
+            oci bastion session get --session-id <managed_session_ocid> \
+                        --query "data.\"ssh-metadata\".command" --raw-output
+        
+        This returns the SSH command needed to access your VM, and will be something similar to:
+
+        .. code::
+
+            ssh -i <privateKey> -o ProxyCommand="ssh -i <privateKey> -W %h:%p -p 22 ocid1.bastionsession.oc1.sa-saopaulo-1.amaaaaaaniwq6syaw7chyowsp55ksemkz67khnd3kprlmwceezuiqe74w4pq@host.bastion.sa-saopaulo-1.oci.oraclecloud.com" -p 22 ubuntu@10.0.1.230
```

Comment:
Same check for the long alphanumeric string here, just want to confirm it's safe and stable